### ISSUE TITLE
"scripture-burrito" in translation

### ISSUE THREAD

**Issue by KenK-3-21** [bible-technology/scripture-burrito#220]
The term "scripture-burrito" occurs in 16 strings in the P9 UI. The term has raised questions in the minds of translators. Though the group who invented the term may feel emotionally attached to it, this term needs to be changed from a colloquial register to a more technical register, in order to be translatable.

See a sample from correspondence with UBS Hong Kong: 

> I came across this string "Wrap project in a Scripture Burrito..." on Crowdin, it occurs many times and I'm not sure what it means and how to translate it into Chinese that users would understand what it means at first glance. Couldn't find it in Paratext/Text Collection. Is this something PT users would see? 
> 
> About the translator's question: if the developer writes "Scripture fried rice" (instead of ... burrito), she will surely know how to translate --- haha.  Just a suggestion, since Paratext has to be localized to many different languages, it would be helpful for the localization teams if the developers could avoid using such "regional language".  So according to the explanation in Crowdin, perhaps something like: "embed the content and metadata into the project".

In my opinion, as a professional translator, and a software localizer, this phrase: "embed the content and metadata into the project" is much more translatable.

**Comment by jonathanrobie** [comment]
The Scripture Burrito Working Group is defining a file format, like Zip files, that can be used for projects.  Scripture Burrito is simply the name of the format. 

In Paratext, we would probably use phrases like "import a project" or "export a project,"  not "embed the content and metadata into the project".   (Paratext does not yet do anything with Scripture Burrito, so we don't need these strings yet, but we will in the coming year.)

But we will have to say that we export into a particular format, and the name of this format is Scripture Burrito - more of a Scripture wonton than a Scripture fried rice. It sounds like you would like to have a noun that is easier to explain across cultures.  Right now, we use the phrase "a Scripture Burrito" to mean "a project archive in Scripture Burrito format".  Are you looking for that kind of phrase?

**Comment by FoolRunning** [comment]
>
> Paratext does not yet do anything with Scripture Burrito
>
This is technically not true. Paratext does have code to generate a SB
(it's what generated the text translation example
<https://github.com/bible-technology/sb_textTranslation>). However, the
menu item to access that export only appears on developer's machines.
Unfortunately, it seems that our string localizer is still finding the
strings related to this. I'll try to do something to hide them for general
builds. The strings can safely be ignored for now.

FWIW, "Scripture Burrito" is a proper name (like "Unicode"). I'm not sure
how they are normally localized, but they should be handled the same way as
other proper nouns. Just something to think about if we do eventually
include it in the UI or help.

*Tim Steenwyk*

*Software developer*
*SIL Language Software Development*
tim_steenwyk@sil.org

**Comment by FoolRunning** [comment]
>
> Paratext does not yet do anything with Scripture Burrito
>
This is technically not true. Paratext does have code to generate a SB (it's what generated the [text translation example](https://github.com/bible-technology/sb_textTranslation)). However, the menu item to access that export only appears on developer's machines. Unfortunately, it seems that our string localizer is still finding the strings related to this. I'll try to do something to hide them for general builds. The strings can safely be ignored for now.

FWIW, "Scripture Burrito" is a proper name (like "Unicode"). I'm not sure how they are normally localized, but they should be handled the same way as other proper nouns. Just something to think about if we do eventually include it in the UI or help.

**Comment by jag3773** [comment]
I agree @FoolRunning that "Scripture Burrito" probably should not be translated and would further suggest that it does not become a menu item that users would see. I would expect something more generic like "Project Import/Export" but of course that's up to you guys.
